data/json/items/comestibles/med.json

```diff
@@ -756,6 +756,18 @@
     "use_action": [ "EYEDROPS" ],
     "flags": [ "NO_INGEST", "NPC_SAFE" ]
   },
+  {
+    "id": "moisturizer",
+    "type": "GENERIC",
+    "name": { "str": "moisturizer" },
+    "description": "A cosmetic preparation used for protecting, moisturizing, and lubricating the skin.",
+    "volume": "50ml",
+    "weight": "50 g",
+    "price": 300,
+    "price_postapoc": 300,
```

Comment:
If it is that valuable post apoc, maybe give it some use? Like a small morale boost if "consumed" or something.

Reply:
It's now worth as much as soap